### PR TITLE
enable p & z assisted installer builds

### PR DESCRIPTION
**Pull request by jacobemery**
Add Power (ppc64le) and Z (s390x) to enable the Assisted Installer to deploy OCP onto these architectures.

The following Assisted Installer components are required to run on the target architecture and are built using the scripts changed in this PR:
- assisted-installer
- assisted-installer-controller
- assisted-installer-agent

Strategy: [OCPPLAN-8327](https://issues.redhat.com/browse/OCPPLAN-8327)
Epic: [MULTIARCH-2889](https://issues.redhat.com/browse/MULTIARCH-2889)
Sub-task: [MGMT-13560](https://issues.redhat.com/browse/MGMT-13560)

### CODE REVIEW

**Comment by jacobemery** [comment]
@droslean @smg247 I believe this is ready to test, could one of you confirm?

**Comment by osherdp** [comment]
> @droslean @smg247 I believe this is ready to test, could one of you confirm?
> 
> Context: https://issues.redhat.com/browse/MGMT-13560

no, their not required for the review. prow sometimes assigns people from admins of the CI system
/uncc droslean smg247
/cc @osherdp @ori-amizur

File: ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master__edge.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ test_binary_build_commands: |
   mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
 tests:
 - as: publish-multi-arch-images
-  postsubmit: true
+  postsubmit: false
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master__edge.yaml:77]: be aware those are temporary changes just to enable testing in this PR. so it should not be merged

File: ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ test_binary_build_commands: |
 tests:
 - as: publish-multi-arch-images
   cluster: build03
-  postsubmit: true
+  postsubmit: false
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml:76]: same as above

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by osherdp** [comment]
@jacobemery you may need to do ``make update`` and push the changes so it can auto-generate job definition changes

**Comment by jacobemery** [comment]
> @jacobemery you may need to do `make update` and push the changes so it can auto-generate job definition changes

Done!

**Comment by jacobemery** [comment]
> it shouldn't really, and you can see the rehearses fail because `quay.io/centos/centos:stream8` doesn't support `s390x` now let's try to only add `ppc64le`

Ah, you're right... Done ^

**Comment by jacobemery** [comment]
> make sure the rehearse jobs are passing. afterwards we can revert the jobs types back from presubmit to postsubmit (and regenerate the code using `make update`)

@osherdp Looks like rehearse tests passed, good to do the above now?

**Comment by osherdp** [comment]
> > make sure the rehearse jobs are passing. afterwards we can revert the jobs types back from presubmit to postsubmit (and regenerate the code using `make update`)
> 
> @osherdp Looks like rehearse tests passed, good to do the above now?

Yes, good to go!

**Comment by jacobemery** [comment]
> Yes, good to go!

Done!

**Review by osherdp — APPROVED** [review]

**Pull request merged**